packages/docsearch-react/src/DocSearchButton.tsx

```diff
@@ -30,13 +30,7 @@ export const DocSearchButton = React.forwardRef<
   }, []);
 
   return (
-    <button
-      type="button"
-      className="DocSearch DocSearch-Button"
-      aria-label="Search"
-      {...props}
-      ref={ref}
-    >
+    <div className="DocSearch DocSearch-Button" {...props} ref={ref}>
       <div className="DocSearch-Button-Container">
```

Comment:
this should be moved to a span instead and the button stays (I think)

Reply:
Yes, we are more and more confident with this button tag.

Reply:
Agree, it makes sense, replaced divs with spans.